migration-engine/migration-engine-tests/tests/native_types/mssql.rs

```diff
@@ -0,0 +1,2260 @@
+            Value::text("<jamon>iberico</jamon>"),
+            &[
+                "TinyInt",
+                "SmallInt",
+                "Int",
+                "BigInt",
+                "Decimal",
+                "Numeric",
+                "Money",
+                "SmallMoney",
+                "Bit",
+                "Float",
+                "Real",
+                "Date",
+                "Time",
+                "DateTime",
+                "DateTime2",
+                "DateTimeOffset",
+                "SmallDateTime",
+                "Text",
+                "NText",
+                "UniqueIdentifier",
+            ],
+        ),
+        (
+            "UniqueIdentifier",
+            Value::text("ce6ae17b-d877-4dbf-85cf-fd0daf0c1385"),
+            &[
+                "TinyInt",
+                "SmallInt",
+                "Int",
+                "BigInt",
+                "Decimal",
+                "Numeric",
+                "Money",
+                "SmallMoney",
+                "Bit",
+                "Float",
+                "Real",
+                "Date",
+                "Time",
+                "DateTime",
+                "DateTime2",
+                "DateTimeOffset",
+                "SmallDateTime",
+                "Char",
+                "NChar",
+                "VarChar",
+                "Text",
+                "NVarChar",
+                "NText",
+                "Binary",
+                "VarBinary",
+                "Image",
+                "Xml",
+            ],
+        ),
+    ]
+});
+
+static TYPE_MAPS: Lazy<HashMap<&'static str, &'static str>> = Lazy::new(|| {
+    let mut maps = HashMap::new();
+
+    maps.insert("TinyInt", "Int");
+    maps.insert("SmallInt", "Int");
+    maps.insert("Int", "Int");
+    maps.insert("BigInt", "BigInt");
+    maps.insert("Decimal", "Decimal");
+    maps.insert("Numeric", "Decimal");
+    maps.insert("Money", "Float");
+    maps.insert("SmallMoney", "Float");
+    maps.insert("SmallMoney", "Float");
+    maps.insert("Bit", "Boolean");
+    maps.insert("Float", "Float");
+    maps.insert("Real", "Float");
+    maps.insert("Date", "DateTime");
+    maps.insert("Time", "DateTime");
+    maps.insert("DateTime", "DateTime");
+    maps.insert("DateTime2", "DateTime");
+    maps.insert("DateTimeOffset", "DateTime");
+    maps.insert("SmallDateTime", "DateTime");
+    maps.insert("Char", "String");
+    maps.insert("NChar", "String");
+    maps.insert("VarChar", "String");
+    maps.insert("Text", "String");
+    maps.insert("NVarChar", "String");
+    maps.insert("NText", "String");
+    maps.insert("Binary", "Bytes");
+    maps.insert("VarBinary", "Bytes");
+    maps.insert("Image", "Bytes");
+    maps.insert("Xml", "String");
+    maps.insert("UniqueIdentifier", "String");
+
+    maps
+});
+
+fn with_params(r#type: &str) -> &str {
+    match r#type {
+        "Decimal" => "Decimal(18,0)",
+        "Numeric" => "Numeric(18,0)",
+        "Float(24)" => "Real",
```

Comment:
just curious, why is this branch different from the others?

Reply:
Because SQL Server renames `Float(24)` to `Real` when you read it back.

Reply:
makes sense, thx